packages/lexure/tests/lexer.test.ts

```diff
@@ -0,0 +1,54 @@
+import { Lexer, QuotedParameter, WordParameter } from '../src';
+
+describe('The lexer tests', () => {
+	it('GIVEN input with no quotes THEN return the string as is', () => {
```

Comment:
We don't use `it()`, we use `test()`, apply this everywhere else as well:
```suggestion
	test('GIVEN input with no quotes THEN return the string as is', () => {
```